### DISCUSSION TITLE
Is it possible to use this tool on EML-L09 (Kirkin 970)

### DISCUSSION THREAD

**Discussion by Decencies** [kitsuned/PotatoNV#121] [Q&A]
I have obtained some (presumably patched) bootloader images, containing the `uce`, `fastboot` & `xloader` images. I have formatted them like all of the other bootloaders, but after verifying and attempting to download, it times out.

```
Verifying images...
Uploading hisi970...
- xloader
- uce
- fastboot
The operation has timed out.
```

Using the latest release of PotatoNV, hopefully there is a working version of this tool for 970 chips.

Here's my zip that I used: [hisi970.zip](https://github.com/mashed-potatoes/PotatoNV/files/8318868/hisi970.zip)

**Comment by byf3332** [comment]
Unfortunately kiirin970 devices have no keys in nvme partition， and hex of the key part on my decvice (mate 10 pro)looks like this`WVLOCKD>�UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDD(Ascii of 57 56 4C 4F 43 4B 00 00 
01 00 00 00 44 00 00 00 
3E 01 1F DD 55 55 55 55 
55 55 55 55 55 55 55 55 
55 55 55 55 55 55 55 55 
55 55 55 55 55 55 55 55 
55 55 55 55 44 44 44 44 
44 44 44 44 44 44 44 44 
44 44 44 44`
I geuss that there is no way using this method :(
